app/assets/javascripts/common/reports.js

```diff
@@ -140,99 +140,111 @@ DashboardReports = () => {
       const config = withBaseLineConfig(
         additionalCumulativeDiabetesRegistrationsTrendConfig
       );
-
+    
       config.options.scales = {
-        xAxes: [
-          {
-            stacked: true,
-            display: true,
-            gridLines: {
-              display: false,
-              drawBorder: false,
-            },
-            ticks: {
-              autoSkip: false,
-              fontColor: colors.darkGrey,
-              fontSize: 12,
-              fontFamily: "Roboto",
-              padding: 8,
-              min: 0,
-              beginAtZero: true,
-            },
+        x: {
+          stacked: true,
+          display: true,
+          grid: {
+            display: false,
+            drawBorder: false,
           },
-        ],
-        yAxes: [
-          {
-            id: "cumulativeDiabetesRegistrations",
-            position: "left",
-            stacked: true,
-            display: true,
-            gridLines: {
-              display: false,
-              drawBorder: false,
-            },
-            ticks: {
-              display: false,
-              autoSkip: false,
-              fontColor: colors.darkGrey,
-              fontSize: 10,
-              fontFamily: "Roboto",
-              padding: 8,
-              min: 0,
-              beginAtZero: true,
-              stepSize: cumulativeDiabetesRegistrationsYAxis.stepSize,
-              max: cumulativeDiabetesRegistrationsYAxis.max,
-              callback: formatNumberWithCommas
+          ticks: {
+            autoSkip: false,
+            color: colors.darkGrey,
+            font: {
+              size: 12,
+              family: "Roboto",
             },
+            padding: 8,
           },
-          {
-            id: "monthlyDiabetesRegistrations",
-            position: "right",
-            stacked: true,
-            display: true,
-            gridLines: {
-              display: true,
-              drawBorder: false,
-            },
-            ticks: {
-              display: false,
-              autoSkip: false,
-              fontColor: colors.darkGrey,
-              fontSize: 10,
-              fontFamily: "Roboto",
-              padding: 8,
-              min: 0,
-              beginAtZero: true,
-              stepSize: monthlyDiabetesRegistrationsYAxis.stepSize,
-              max: monthlyDiabetesRegistrationsYAxis.max,
-              callback: formatNumberWithCommas
+          beginAtZero: true,
+          min: 0,
+        },
+
+        // temporary fix for hiding y axis
+        y: {
+          display: false,
+        },
+
+        yCumulativeDiabetesRegistrations: {
```

Comment:
I know we used to do this too, but can you explain to me why we choose to set a custom ID for the y-axis and use that instead of the default?

Reply:
Each dataset added in the current setup adds a new Y axis.
Each Y axis name must match exactly the yAxisID stated in the dataset config:
```js
datasets: [
            {
              yAxisID: "yCumulativeDiabetesRegistrations",
              ...
            },
            {
              yAxisID: "yMonthlyDiabetesFollowups",
              ...
            },
            {
              yAxisID: "yMonthlyDiabetesRegistrations",
              ...
            },
```
(Scroll up to line 104 above)
This means I need change the y-axis object names to match yAxisID's.
Under the current setup the first dataset added should always be y then any other datasets will be named - this only occurs on the registration line graphs.
I have to hide the Y axis on this occasion as the default config means that it shows (even with no data present).
Its a plaster over the issue - but in the next DRY PR I have addressed this by merging axes.
Again this is purely a PR to push v3.9.1 over the line and I have comeback to remove config under the new structure in the next PR 😅 

Does this answer your question?